src/Calculator.jsx

```diff
@@ -0,0 +1,189 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const Container = styled.section`
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+  align-items: center;
+  color: #383854;
+  margin: 2rem 0 2rem 0;
+  font-size: 2rem;
+  @media only screen and (max-width: 420px) {
+    margin: 0.25rem 0;
+  }
+`;
+
+const SubTitle = styled.h2`
+  font-size: 1rem;
+  width: 50%;
+  color: #383854;
+  @media only screen and (max-width: 700px) {
+    font-size: 0.75rem;
+  }
+  @media only screen and (max-width: 420px) {
+    width: 75%;
+  }
+`;
+
+const Activitys = styled.section`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  font-size: 1rem;
+  color: #383854;
+  max-width: 50%;
+  border-radius: 1rem;
+
+  @media only screen and (max-width: 700px) {
+    align-items: center;
+    padding: 1rem 1rem;
+  }
+
+  @media only screen and (max-width: 420px) {
+    padding: 0;
+  }
+`;
+
+const IconContainer = styled.section`
+  display: flex;
+  justify-content: center;
+  margin: 1rem 0 0 0;
+
+  @media only screen and (max-width: 420px) {
+    margin: 0;
+  }
+`;
+
+const Icon = styled.button`
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  border-radius: 1rem;
+  color: white;
+  background-color: #383854;
+  border: 0.05rem rgba(255, 255, 255, 0.75) solid;
+  box-shadow: 0.2rem 0.2rem 0.3rem #888888;
+  margin: 0 0.5rem;
+  padding: 2rem 2rem;
+
+  @media only screen and (max-width: 700px) {
+    padding: 1rem 1rem;
+  }
+  @media only screen and (max-width: 420px) {
+    margin: 0.5rem;
+  }
+`;
+
+const Img = styled.img`
+  width: 2rem;
+  height: 2rem;
+`;
+
+const Calculate = styled.button`
+  display: flex;
+  justify-content: center;
+  font-size: 1rem;
+  font-weight: bold;
+  color: #383854;
```

Comment:
I have already given my opinion on this button being a different colour! It is a call to action and I think having a more stand out colour makes sense here. The [google material design docs](https://material.io/guidelines/) are useful as they lay out some styling guidelines.

Reply:
i'll work on that